dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/persistence/file/FileOrchestrator.kt

```diff
@@ -32,6 +32,11 @@ internal interface FileOrchestrator {
      */
     fun getReadableFile(excludeFiles: Set<File>): File?
 
+    /**
+     * @return a List of all flushable files
```

Comment:
maybe we need to define what is the flushable files and what is the use-case for this method? wdyt?

Reply:
I will add more comments there